package/gluon-status-page/javascript/status-page.js

```diff
@@ -444,8 +444,8 @@
 	}
 
 	function Neighbour(iface, addr, color, destroy) {
-		var th = iface.table.firstElementChild;
-		var el = iface.table.insertRow();
+		var th = iface.table.firstElementChild.firstElementChild;
+		var el = iface.table.firstElementChild.insertRow();
```

Comment:
We can avoid changing this code by assigning `el.firstElementChild.firstElementChild` to `info.table` in `Interface()`.

Reply:
That would then imho change the meaning of the code, i would prefer some way of
```
var tb = iface.table.firstElementChild;
var th = tb.firstElementChild;
var el = tb.insertRow();
```
which would make it clearer.

The better way would even be here to select the elements via css selectors.

Reply:
Well, or
```js
var th = iface.tbody.firstElementChild;
var el = iface.tbody.insertRow();
```
removing the field `table` from `iface`, and assigning `tbody` instead. As far as I can tell, nothing else uses the field `table`.

Reply:
I'm also fine with using more CSS selectors instead of all that `firstElementChild`, but that's mostly orthogonal to the question which elements to reference in the JS data structures.

Reply:
I added css selectors in the last commit, feel free to remove it before merging :D